MetaMask/metamask-extension
# feat: add bifrost logo svg

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/29340?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2. Add Bifrost Polkadot Mainnet
3. Bifrost Polkadot Mainnet should have a logo

## **Screenshots/Recordings**

### **Before**

<img width="359" alt="01" src="https://github.com/user-attachments/assets/68ae336d-e134-49cc-8a5e-b4fd48b4e7c7" />

### **After**

<img width="359" alt="02" src="https://github.com/user-attachments/assets/b012c4a8-a518-4d97-a8d9-a37fef56c323" />


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
